### PR TITLE
Gateways

### PR DESCRIPTION
**Pull request by qbalukom**
**Description**
Implement a list and details of istio gateways

### CODE REVIEW

**Review by NHingerl — DISMISSED** [review]

**Review by Wawrzyn321 — DISMISSED** [review]
Looks nice, do you think adding the `selectors` in details header (maybe also on list?) would be useful for the users?

File: core/src/luigi-config/navigation/static-navigation-model.js [hunk 1]
```diff
@@ -1059,6 +1059,43 @@ export function getStaticChildrenNodesForNamespace(
         },
       ],
     },
+    {
+      category: i18next.t('configuration.title'),
+      resourceType: 'gatewats',
```
Comment (akucharska) [inline core/src/luigi-config/navigation/static-navigation-model.js:1064]: Shouldn't these be gateways?

File: core/src/luigi-config/navigation/static-navigation-model.js [hunk 2]
```diff
@@ -1059,6 +1059,43 @@ export function getStaticChildrenNodesForNamespace(
...
+      pathSegment: 'gateways',
+      label: i18next.t('gateways.title'),
+      viewUrl:
+        config.coreUIModuleUrl +
+        '/namespaces/:namespaceId/gateways?' +
+        toSearchParamsString({
+          resourceApiPath: '/apis/networking.istio.io/v1beta1',
```
Comment (akucharska) [inline core/src/luigi-config/navigation/static-navigation-model.js:1071]: Shouldn't these be `/apis/networking.istio.io/v1alpha3`

File: core/src/luigi-config/navigation/static-navigation-model.js (hunk 1 above)
Comment (qbalukom) [reply core/src/luigi-config/navigation/static-navigation-model.js]: they most certainly should

File: core-ui/src/components/Predefined/Details/Gateway/GatewayServers.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    ? `${port.number}:${port.targetPort}`
+    : port.number;
+  return `${port.name} (${number})`;
+}
+
+function ServerHosts({ hosts }) {
+  return (
+    <ul>
+      {hosts.map(host => (
+        <li>{host}</li>
```
Comment (akucharska) [inline core-ui/src/components/Predefined/Details/Gateway/GatewayServers.js:16]: add key

File: core-ui/src/components/Predefined/Details/Gateway/GatewaySelector.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Token } from 'fundamental-react';
+
+export function GatewaySelector({ gateway }) {
+  return Object.entries(gateway.spec.selector).map(([key, value]) => (
+    <Token readOnly key={key}>
```
Comment (akucharska) [inline core-ui/src/components/Predefined/Details/Gateway/GatewaySelector.js:6]: Add  `buttonLabel`

**Review by akucharska — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**